python/ray/serve/api.py

```diff
@@ -910,41 +919,6 @@ def remote(self, *args, **kwargs) -> ray.ObjectRef:
         raise NotImplementedError()
 
 
-@PublicAPI(stability="alpha")
-class DeploymentMethodNode(DAGNode):
-    """Represents a method call on a bound deployment node.
-
-    These method calls can be composed into an optimized call DAG and passed
-    to a "driver" deployment that will orchestrate the calls at runtime.
-
-    This class cannot be called directly. Instead, when it is bound to a
-    deployment node, it will be resolved to a DeployedCallGraph at runtime.
-    """
-
-    def __init__(self):
-        raise NotImplementedError()
-
-
-@PublicAPI(stability="alpha")
-class DeploymentNode(DAGNode):
-    """Represents a deployment with its bound config options and arguments.
-
-    The bound deployment can be run, deployed, or built to a production config
-    using serve.run, serve.deploy, and serve.build, respectively.
```

Comment:
@jiaodong can you please move the class definitions to where they were originally defined in the interfaces PR? The diff is kinda messed up, it's interleaving multiple class definitions